bundles/net.tourbook.export/src/net/tourbook/export/DialogExportTour.java

```diff
@@ -1342,7 +1341,10 @@ private void doExport_21_Creator(final VelocityContext vcContext) {
          pluginMajorVersion = Integer.toString(version.getMajor());
          pluginMinorVersion = Integer.toString(version.getMinor());
          pluginMicroVersion = Integer.toString(version.getMicro());
-         pluginQualifierVersion = version.getQualifier();
+         final String versionQualifier = version.getQualifier();
+         if (StringUtils.isNumeric(versionQualifier)) {
```

Comment:
That's why I am testing if it's a numeric because the Garmin XSD schema expects an "unsignedshort"
<xsd:element name="BuildMinor" type="xsd:unsignedShort" minOccurs="0"/>
[Source](https://www8.garmin.com/xmlschemas/TrainingCenterDatabasev2.xsd)